src/App.css

```diff
@@ -36,3 +36,66 @@
     transform: rotate(360deg);
   }
 }
+
+.header-container {
+    display: flex;
+    width: 100%;
+    height: 60px;
+    background-color: #2e2e2e;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 10px;
+}
```

Comment:
We must keep global styling at its minimum and put styles together with the component they belong